spec/features/registering_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require 'tilt/erb'
+
+feature 'Registering,' do
+	scenario 'User can sign up as a new user' do
+		expect{sign_up}.to change(User, :count).by(1)
+		expect(page).to have_content('Welcome, d9nny')
+		expect(User.first.email).to eq('dan@google.com')
```

Comment:
This is a little bit of a magic string - where did 'dan@google.com' come from? If you allowed your sign up method to accept an argument, that would show that you are entering 'dan@google.com', and make it clear that you are expecting that to be passed into the database. 

Also - this is not testing user behaviour, rather you are testing the model directly, which does not really belong in the feature test.